### DISCUSSION TITLE
Dashboard ?

### DISCUSSION THREAD

**Discussion by WeenyVMS** [dimaguy/ImpostorHQ#17] [Q&A] [answered]
Hey, I don't understand how the dashboard works

**Comment by dimaguy** [comment, marked as answer]
This plugin is composed of 4 components, a websocket api, an http server, an extension api and the frontend(the dashboard itself).  

The dashboard frontend is served over the provided http server, it connects to the Impostor server using the websocket api which provides a pipe to send and receive data(commands, console, graphs etc).

Commands are provided through a default set of commands and other extensions.

Theres a tad more to this but this is the basic gist of it.
